addon/components/sl-select.js

```diff
@@ -199,19 +193,19 @@ export default Ember.Component.extend( InputBased, TooltipEnabled, {
                     Ember.warn( warning, !values.length );
                 }
 
-                return callback( multiple ? filteredContent : filteredContent.get( 'firstObject' ) );
+                return callback( multiple ? filteredContent: Ember.get( filteredContent, 'firstObject' ) );
             },
 
-            minimumResultsForSearch: this.get( 'disableSearch' ) ? -1 : 0,
+            minimumResultsForSearch: this.get( 'disableSearch' ) ? -1: 0,
 
             query: function( query ) {
-                var content         = self.get( 'content' ) || [],
+                var content = self.get( 'content' ) || [],
                     optionLabelPath = self.get( 'optionLabelPath' ),
-                    select2         = this;
+                    select2 = this;
 
                 query.callback({
                     results: content.reduce( function( results, item ) {
-                        var text = item instanceof Object ? get( item, optionLabelPath ) : item;
+                        var text = item instanceof Object ? get( item, optionLabelPath ): item;
```

Comment:
Do we want to remove this space?  This change does not seem consistent with other usages.